src/docs/markdown/caddyfile/options.md

```diff
@@ -446,6 +446,9 @@ The name to put in the CommonName field of the root certificate. Default: `{pki.
 ##### `intermediate_cn`
 The name to put in the CommonName field of the intermediate certificates. Default: `{pki.ca.name} - ECC Intermediate`
 
+##### `intermediate_lifetime`
```

Comment:
Should also add this to the example/syntax block at the top of the file

Reply:
Updated 🙂